packages/infra/config/env-config.ts

```diff
@@ -82,6 +81,10 @@ export type EnvConfig = {
     CW_GATEWAY_AUTHORIZATION_CLIENT_ID: string;
     CW_GATEWAY_AUTHORIZATION_CLIENT_SECRET: string;
   };
+  ihe: {
+    CERT_ARN: string;
```

Comment:
nit: we usually have all caps for env vars and secret names, in this case its just a regular value/property so it cold use camel case